test/integration/dialects/mssql/regressions.test.js

```diff
@@ -249,5 +250,28 @@ if (dialect.startsWith('mssql')) {
       const record = await BooleanTable.findOne();
       expect(record.status).to.equals(value);
     });
+
+    it('rolls back changes after inserting more than 1000 rows outside of a transaction', async function () {
```

Comment:
I would add a link to the task that describes the problem here, future maintainers (even if it's us) could think this test is redundant